WordPress/two-factor
# Update two factor options layout

- Update and move two factor option heading out of table
- Align two factor options with title

## What?

This PR updates the layout of Two Factor options to match Application Passwords, fixes https://github.com/WordPress/two-factor/issues/622

## Why?

To make sure the UI consistent, as mentioned in https://github.com/WordPress/two-factor/issues/622, the request in it is to use div tags but we can't do it as `fieldset` tag is required to make all inputs disabled on a condition, which won't work with `div` without the some additional CSS.

## How?

- It moves the `Two-Factor Options` out of the table and adds a heading for it, adds some style to align the two factor options table with heading.

## Testing Instructions

- Start the env after checking out into the branch go  to http://localhost:8888/wp-admin/profile.php, scroll down two factor options, confirm it matches the Application Passwords.

## Screenshots or screencast

![CleanShot 2024-08-16 at 17 33 31@2x](https://github.com/user-attachments/assets/cab3ed99-e213-442a-99ec-d67327f961c2)


## Changelog Entry
> Changed - Two Factor Options layout
